api/src/wfl/service/google/bigquery.clj

```diff
@@ -1,13 +1,11 @@
 (ns wfl.service.google.bigquery
   "Wrappers for Google Cloud BigQuery REST APIs.
    See https://cloud.google.com/bigquery/docs/reference/rest."
-  (:require [clj-http.client :as http]
-            [clojure.data.csv :as csv]
-            [clojure.java.io :as io]
+  (:require [clj-http.client   :as http]
             [clojure.data.json :as json]
-            [clojure.string :as str]
-            [wfl.auth :as auth]
-            [wfl.util         :as util])
+            [clojure.string    :as str]
+            [wfl.auth          :as auth]
+            [wfl.util          :as util])
```

Comment:
sad :'(

Reply:
My IDE does this alignment too (and I quite like it tbh :P)

Reply:
Ah, you know I think I might have introduced this alignment myself because I saw it done in someone else's PR!

For future reference, do we have a preference here?